client/app/lib/service.js

```diff
@@ -116,7 +116,7 @@ export const getQuery = async ({ id }) => {
 
 export const executeQuery = async (payload) => {
   try {
-    const res = await axios.post(`${QUERY_EXECUTION}`, payload);
+    const res = await axios.get(`${QUERY_EXECUTION}?db_id=${payload.db_id}&sql_query=${payload.sql_query}`);
```

Comment:
You can use `URLSearchParams` to convert the query parameters into a query string before passing it to `axios.get`. This approach is cleaner and more manageable.